crates/cheatcodes/assets/cheatcodes.json

```diff
@@ -6391,6 +6391,66 @@
       "status": "stable",
       "safety": "safe"
     },
+    {
+      "func": {
+        "id": "randomAddress",
+        "description": "Returns a random `address`.",
+        "declaration": "function randomAddress() external returns (address);",
+        "visibility": "external",
+        "mutability": "",
+        "signature": "randomAddress()",
+        "selector": "0xd5bee9f5",
+        "selectorBytes": [
+          213,
+          190,
+          233,
+          245
+        ]
+      },
+      "group": "utilities",
+      "status": "stable",
+      "safety": "safe"
+    },
+    {
+      "func": {
+        "id": "randomUint_0",
+        "description": "Returns a random uint256 value.",
```

Comment:
Could we document whether the value is from a CSPRNG or not, ie whether its cryptographically secure randomness? If it is not, people may use it to create insecure private keys. If it is, auditors will have to jump into foundry's source to verify.

Reply:
Great call out, this + https://github.com/foundry-rs/foundry/pull/7960#discussion_r1619015391 should be added to the foundry book